examples/rcn/load_data.py

```diff
@@ -0,0 +1,78 @@
+# export
+import os
+
+import cv2
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# export
+def get_mnist_data_iters(data_dir, train_size, test_size, full_test_set=False, seed=5):
```

Comment:
As discussed, replace this function with something that can download the MNIST datasets. It seems https://pypi.org/project/tensorflow-datasets/ is a standalone package that works well with JAX, so I suggest we add this to our dependency (by updating https://github.com/vicariousinc/PGMax/blob/master/pyproject.toml) and use it to access MNIST